test/svg-sprite.js

```diff
@@ -131,21 +160,29 @@ function compareSvg2Png(svg, png, expected, diff, done, msg) {
         done();
     };
     fs.readFile(svg)
-        .then(svg2png)
-        .then(function (buffer) {
-            fs.writeFile(png, buffer)
-                .then(function () {
-                    imageDiff({
-                        actualImage: png,
-                        expectedImage: expected,
-                        diffImage: diff
-                    }, function (err, imagesAreSame) {
-                        should(err).not.ok;
-                        should.ok(imagesAreSame, msg);
-                        done();
-                    });
-                })
-                .catch(ecb);
+        .then(function(buffer) {
+            sharp(buffer).toFile(png, function(err) {
```

Comment:
I was wondering, can't we use `readFile` without an `enconding` set (thus returning a Buffer) and `writeFile`? Do we need sharp?